Doc/library/pathlib.rst

```diff
@@ -922,6 +922,9 @@ call fails (for example because the path doesn't exist):
    *target* exists and is a file, it will be replaced silently if the user
    has permission.  *target* can be either a string or another path object::
 
+   If the provided *target* is a path object, it is the method's return value.
+   Otherwise (if it is a string), new path object is created from it and returned.
+
```

Comment:
The example below needs to be updated now that `p.rename(target)` is returning a value (double-check this in a terminal):

```python
>>> p = Path('foo')
>>> p.open('w').write('some text')
9
>>> target = Path('bar')
>>> p.rename(target)
Path('bar')
>>> target.open().read()
'some text'
```